style.css

```diff
@@ -82,76 +87,241 @@ a:hover {
   color: #e31c3a;
 }
 
-img {
-  width: 100%;
+.big-meal-photo {
+  width: 70%;
   height: auto;
+  margin: 0 auto;
 }
 
 .input-button-field {
   display: flex;
   flex-direction: column;
+  width: 100%;
+  margin: 0 auto;
 }
 
 .email {
-  margin-top: 2em;
+  margin-bottom: 1em;
   border: 1px solid #e31c3a;
-  border-radius: 4px;
+  border-radius: 0.25em;
   background-color: #ffe6ee;
   padding: 0.8em;
   color: #080808;
   outline: none;
 }
 
 .button {
-  margin: 1em 0;
+  margin-bottom: 1em;
   border: 1px solid #e31c3a;
-  border-radius: 4px;
+  border-radius: 0.25em;
   background-color: #e31c3a;
   padding: 0.8em;
   color: #ffffff;
   outline: none;
   cursor: pointer;
 }
 
+.section-two {
+  width: 100%;
+  padding: 0 1em;
+  margin-bottom: 3em;
+}
+
+.head-meal-section {
+  text-align: center;
+  margin-bottom: 0.5em;
+  line-height: 1em;
+  font-size: 2.2em;
+}
+
+.under-head-meal-section {
+  text-align: center;
+  font-size: 0.9em;
+  margin-bottom: 2em;
+}
+
+.about-you {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-bottom: 2em;
+}
+
+.vege-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.rotateimg180 {
+  transform: rotate(90deg);
+}
+
+.title-about-you {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  text-align: center;
+  margin: 1em 0em;
+}
+
+.vege-text {
+  font-size: 0.75em;
+  margin-top: 1em;
+  text-align: center;
+}
+
+.meals {
+  display: flex;
+  flex-direction: column;
+}
+
+.refugee-block {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2em;
+}
+
+.refugee-title {
+  color: #0a0a0a;
+  font-size: 2.2em;
+  text-align: center;
+  line-height: 1em;
+  margin-bottom: 1em;
+}
+
+.refugee-text {
+  font-size: 0.75em;
+  text-align: center;
+  margin-bottom: 1em;
+}
+
+.meal-block {
+  display: flex;
+  margin-bottom: 2em;
+}
+
+.meal-image {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```

Comment:
Just a general suggestion here: In the future, you could use something like `flex-center-center` class, which will have the corresponding CSS rules, so you don't have to repeat yourself (DRY principle) everywhere you plan to center an element (or else).

P.S.
Don't need to change anything here, just for future reference 😄 

Keep in mind that the CSS frameworks (e.g. Bootstrap) use something like I mentioned above 🥇 

Adding @dsekulovic to read this hint, as well.